### PR TITLE
Use isodatetime-2017.02.1

### PR DESCRIPTION
**Pull request by matthewrmshin**

### CODE REVIEW

**Comment by matthewrmshin** [comment]
(Travis CI is a bit broken at the moment, telling us that it cannot install some standard stuffs.)

File: lib/isodatetime/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#-----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
 # (C) British Crown Copyright 2013-2014 Met Office.
```
Comment (oliver-sanders) [inline lib/isodatetime/__init__.py]: Happy new year * 3.

Comment (matthewrmshin) [inline lib/isodatetime/__init__.py]: Actually, I have forgotten to do this for this isodatetime release. I'll get that done tomorrow.

Comment (matthewrmshin) [inline lib/isodatetime/__init__.py]: Now 3 times happy.

**Pull request merged**